packages/runtime/container-runtime/src/test/containerRuntime.spec.ts

```diff
@@ -2368,11 +2369,14 @@ describe("Runtime", () => {
 		});
 
 		describe("Duplicate Batch Detection", () => {
-			for (const enableOfflineLoad of [undefined, true]) {
-				it(`DuplicateBatchDetector enablement matches Offline load (${enableOfflineLoad === true ? "ENABLED" : "DISABLED"})`, async () => {
+			for (const enableDuplicateBatchDetection of [undefined, true]) {
+				it(`DuplicateBatchDetector enablement matches Offline load (${enableDuplicateBatchDetection === true ? "ENABLED" : "DISABLED"})`, async () => {
```

Comment:
For context: This test is highlighting the fact that Duplicate Batch Detection is controlled by the one setting that controls the other BatchId related stuff.  Rather than being independently controlled.